client/src/components/SeasonCollectionImageCardComponent/SeasonCollectionImage.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import imageFile1 from "../SeasonCollectionImageCardComponent/SeasonCollectionAssetDirectory/Image 1.svg";
+import imageFile2 from "../SeasonCollectionImageCardComponent/SeasonCollectionAssetDirectory/Image 2.svg";
+
+import SeasonCollectionImageCardComponent from "./SeasonCollectionImage";
+
+storiesOf("SeasonCollectionImageCardComponent", module).add(
+  "SeasonCollection",
+  () => (
+    <SeasonCollectionImageCardComponent
+      image1={imageFile1}
+      image2={imageFile2}
```

Comment:
Could you make this an array? If you find yourself adding numbers to variables, then make an array! Never have numbers in your variable names